content/sensu-go/5.12/reference/agent.md

```diff
@@ -1202,6 +1203,41 @@ required              | false
 type                  | Boolean
 example               | {{< highlight shell >}}"enable_env": true{{< /highlight >}}
 
+### Configuration via environment variables
+
+The `sensu-agent` service configured by our supported packages will read environment variables from `/etc/default/sensu-agent` on Debian/Ubuntu systems and `/etc/sysconfig/sensu-agent` on RHEL systems. These files are not created by the installation package and will need to be created.
+
+{{< language-toggle >}}
+
+{{< highlight "Ubuntu/Debian" >}}
+$ sudo touch /etc/default/sensu-agent
+{{< /highlight >}}
+
+{{< highlight "RHEL/CentOS" >}}
+$ sudo touch /etc/sysconfig/sensu-agent
+{{< /highlight >}}
+
+{{< /language-toggle >}}
+
+For any configuration flag you wish to specify as an environment variable, you will need to append `sensu_` and convert any dashes (`-`) to underscores (`_`). The resulting variable will then need to be added to the appropriate environment file described above; the service will need to be restarted before these settings will take effect.
+
+In the following example `api-host` flag is configured as an environment variable and set to `"0.0.0.0"`.
+
+{{< language-toggle >}}
+
+{{< highlight "Ubuntu/Debian" >}}
+$ echo 'sensu_api_host="0.0.0.0' | sudo tee -a /etc/default/sensu-agent
+$ sudo systemctl restart sensu-agent
+{{< /highlight >}}
+
+{{< highlight "RHEL/CentOS" >}}
+$ echo 'sensu_api_host="0.0.0.0' | sudo tee -a /etc/sysconfig/sensu-agent
+$ sudo systemctl restart sensu-agent
+{{< /highlight >}}
+
+{{< /language-toggle >}}
+
+
 ### Example allow list configuration file
```

Comment:
Please move the allow list configuration example above this section so it stays near the allow list configuration specifications. I believe this will also require rearranging the table of contents entries.